examples/next/README.md

```diff
@@ -0,0 +1,15 @@
+# Next Example
```

Comment:
I think this example needs much better documentation. What is the purpose of all these different files? Why did it take 20 different files to demonstrate this feature? Is this it reasonable for a user of this library to see all these files and absorb what each one is doing and figure out to implement the feature in their application within 20-30 minutes?

It should be.

Reply:
Next requires a bunch more work and customization due to having to use getInitialProps.

We could build a library to supply these next specific HOC's, next-react-feature-toggles. I guess they could come from this library too, but I was avoiding making this specific to next at the start.